spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -943,2259 +807,1399 @@ header, and the mode is OUT.
+  use = SOAPBinding.Use.LITERAL,
+  parameterStyle = SOAPBinding.ParameterStyle.WRAPPED) 
+public class DocWrappedService {
+
+   @WebMethod(operationName = "SubmitPO")
+   @WebResult(name="PurchaseOrderAck")
+   public PurchaseOrderAck submitPO(
+       @WebParam(name="PurchaseOrder") PurchaseOrder purchaseOrder) {
+   }
+}
+----
 
-<input>
+*Resulting WSDL:*
 
-<soap:body parts="arg0" use="literal"/>
+[source,xml]
+----
+<definitions
+  xmlns="http://schemas.xmlsoap.org/wsdl/"
+  xmlns:tns="http://www.openuri.org/jwsm/SoapBindingExample3"
+  xmlns:s="http://www.w3.org/2001/XMLSchema"
+  xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/"
+  targetNamespace="http://www.openuri.org/jwsm/SoapBindingExample3">
+
+  <types>
+    <s:schema elementFormDefault="qualified" targetNamespace="http://www.openuri.org/jwsm/SoapBindingExample3">
+       <s:element name="SubmitPO">
+         <complexType>
+           <sequence>
+             <element name="PurchaseOrder" type="tns:PurchaseOrder"/>
+                . . .
+       </s:element>
+
+       <s:element name="SubmitPOResponse">
+           . . .
+       </s:element>
+       
+    </s:schema>
+  </types>
+  
+  <message name="SubmitPO">
+     <part name="parameters" element="tns:SubmitPO"/>
+  </message>
+  
+  <message name="SubmitPOResponse">
+     <part name="parameters" type="tns:SubmitPOResponse"/>
+  </message>
+
+  <portType name="DocWrappedService">
+     <operation name="SubmitPO">
+        <input message="tns:SubmitPO"/>
+        <output message="tns:SubmitPOResponse"/>
+     </operation
+  </portType>
+  
+  <binding name="ExampleServiceHttpSoap" type="ExampleService">
+    <soap:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+    <operation name="SubmitPO">
+      <soap:operation soapAction="http://www.openuri.org/jwsm/SoapBindingExample3/SubmitPO" />
+      <input>
+        <soap:body parts="parameters" use="literal"/>
+      </input>
+      <output>
+        <soap:body parts="parameters" use="literal"/>
+      </output>
+    </operation>     
+  </binding>
+</definitions>
+----
 
-<soap:header message="SecurePing" part="arg1"
+=== Annotation: javax.jws.soap.SOAPMessageHandlers
 
-use="literal"/>
+*Deprecated* as of Jakarta Web Services Metadata 2.0 with no replacement.
 
-</input>
+This annotation was originally used to create a Jakarta XML RPC handler chain.
+In this version, the annotation is ALWAYS ignored.
 
-</operation>
+This annotation will be permanently removed in a future version of
+Jakarta Web Services Metadata.
 
-</binding>
+== Java Mapping To XML/WSDL
 
-</definitions>
+A key goal of Jakarta Web Services Metadata is to influence the shape of WSDL generated from a
+JWS. This section defines the mapping from Java to XML/WSDL. By default,
+Jakarta Web Services Metadata follows the Java to XML/WSDL mapping defined in Jakarta XML Web Services 2.3 [5]
+section 3), except as noted in this section. Implementations MAY extend
```

Comment:
"...defined in Jakarta XML Web Services Specification [5] section 3,..." (there is offending ')' character)

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/62fdf397f28a148826367da36bcaef2c30e97c7d